doc/minversions.rst

```diff
@@ -15,62 +15,64 @@ the rest of the Scientific Python ecosystem, and therefore:
       - Documentation
       - GMT
       - Python
-      - NumPy
+      - Core package dependencies
```

Comment:
In https://github.com/GenericMappingTools/pygmt/pull/3037#discussion_r1479035637, @weiji14 proposed to add two more columns for pandas and xarray. I think this is not good, because the table is already too wide. I prefer to list the core dependencies in a single column. See https://pygmt-dev--3043.org.readthedocs.build/en/3043/minversions.html for preview.

I also prefer to combine the "Python version" and "Documentation" columns into a single column. The new table will look like:

<img width="581" alt="image" src="https://github.com/GenericMappingTools/pygmt/assets/3974108/3945d86a-4dd7-4aa8-9e62-12b1a1bdee8e">

Thoughts?

Reply:
> I also prefer to combine the "Python version" and "Documentation" columns into a single column.

Yep, ok with this part. We can add a note at the bottom to say that 'Dev' reflects the main branch and is for the upcoming release.

> In [#3037 (comment)](https://github.com/GenericMappingTools/pygmt/pull/3037#discussion_r1479035637), @weiji14 proposed to add two more columns for pandas and xarray. I think this is not good, because the table is already too wide. I prefer to list the core dependencies in a single column.

Once we combine the PyGMT and Docs columns, maybe there's enough space for separate numpy/pandas/xarray columns? Could you try splitting into individual columns to see how it looks?

Reply:
Done in 5f22ecb0d86be78634a39de8bcf91c9378ed60cb. Not bad. See preview at https://pygmt-dev--3043.org.readthedocs.build/en/3043/minversions.html.